Sources/Target/Common/ProcessBase.cpp

```diff
@@ -198,7 +198,7 @@ ErrorCode ProcessBase::enumerateMappedFiles(
   // TODO: this is only looking for modules loaded in memory, not all the
   // possible files
   return enumerateSharedLibraries([&](SharedLibraryInfo const &library) {
-    cb({library.path, library.sections[0]});
+    cb({library.path, library.sections[0], library.sizeOfImage});
```

Comment:
Also, sizeOfImage won't work for non-Windows platforms I think because the mappings are non-contiguous. You should probably remove `sizeOfImage` from the `SharedLibraryInfo` structure (but keep it in `MappedFileInfo`), and just pass `-1` here directly to the constructor of `MappedFileInfo`).